test/Icons.Test/Services/IconFetchingServiceTests.cs

```diff
@@ -6,6 +6,7 @@ namespace Bit.Icons.Test.Services;
 public class IconFetchingServiceTests : ServiceTestBase<IconFetchingService>
 {
     [Theory]
+    [InlineData("www.twitter.com")] // https site
```

Comment:
❓ Anything special with how this test was handled? Seems the same as the line below.

Reply:
From when I stepped through the retrieval it's the same as google, we have just been getting reports that twitter in particular is having problems.

Because this work is so pervasive I have no clue if this test passing represents a functionality change or not, but I wanted this as a canary. If it proves to be flaky on twitter's end I want to remove.